wordpress-mobile/WordPress-Android
# Issue/11267 aztec dark mode

Fixes #11267

This PR depends on #11252, so it needs to be merged first.

This PR adds a Dark Mode support to Aztec editor.

Aztec is going to be replaced by Block editor at some point, so I decided not to spend too much time playing with the original library, and instead just override colors for dark mode.

I have only a rough idea of colors used in Dark Mode for Aztec, so some of the parts are improvised.
 
![comparison_aztec](https://user-images.githubusercontent.com/728822/74076322-4c457d80-49cc-11ea-9567-439d8fa03f64.png)


To test:
- Try different functions of the Aztec editor, and make sure it looks good in Dark Mode.


PR submission checklist:
